test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -12,7 +12,7 @@ export function setup(opts: minimist.ParsedArgs) {
 		beforeSuite(opts);
 		afterSuite(opts);
 
-		it('can perform basic text cell functionality', async function () {
+		it.skip('can perform basic text cell functionality', async function () {
```

Comment:
@lewis-sanchez Do we have an issue created to track fixing/re-enabling these skipped tests?

Reply:
I made an issue: https://github.com/microsoft/azuredatastudio/issues/24001